ui/component/markdownLink/view.jsx

```diff
@@ -114,7 +112,7 @@ function MarkdownLink(props: Props) {
         uri={lbryUrlFromLink || decodedUri}
         autoEmbed={embed}
         parentCommentId={parentCommentId}
-        isMarkdownPost={isMarkdownPost}
+        isMarkdownPost
```

Comment:
For `MarkdownLink`, is there still different handling needed between Post and non-Post (e.g, comment, livestream, descriptions, etc.)?

_(I vaguely recall it was needed for the Staking thing (show or not show) ... but surrounding code looked different now, so not sure.)_

- If yes, then `isMarkdownPost` shouldn't be removed from the prop list, at least for future use.
- If no, then remove from the 1 remaining instance:

```
            <MarkdownLink
              {...linkProps}
              parentCommentId={parentCommentId}
              isMarkdownPost={isMarkdownPost}  <<<< HERE
              simpleLinks={simpleLinks}
              allowPreview={isStakeEnoughForPreview(stakedLevel) || hasMembership}
              setUserMention={setUserMention}
            />
```

Reply:
I think there shouldn't be any different handling ? Because all it does is set `source: 'markdown'` on `PlayingUri` so either way, the source of playing should still be markdown from `<MarkdownLink />`, then comments pass `parentCommentId`, so comment & livestream chat will have `source: 'comment'` and descriptions & posts `source: 'markdown'`